src/main/java/ch/njol/skript/expressions/ExprEntities.java

```diff
@@ -79,9 +85,14 @@ public class ExprEntities extends SimpleExpression<Entity> {
 	private Expression<Number> radius;
 	@Nullable
 	private Expression<Location> center;
+	@Nullable
+	private Expression<Location> from;
+	@Nullable
```

Comment:
These should probably all be UndefinedNullability

Reply:
It says that annotation is deprecated. Is this still an important change?

Reply:
UnknownNullability, sorry